src/common/utility.c

```diff
@@ -679,9 +679,6 @@ gchar *dt_util_normalize_path(const gchar *_input)
   // on Windows filenames are case insensitive, so we can end up with an arbitrary number of different spellings for the same file.
   // another problem is that path separators can either be / or \ leading to even more problems.
 
-  // TODO:
-  // this only handles filenames in the old <drive letter>:\path\to\file form, not the \\?\UNC\ form and not some others like \Device\...
-
```

Comment:
Seems like current code added support only for `\\host\path` (UNC) paths, not `\Device\path` paths so I'd leave the `\Device\` part of todo.

seems like it's least popular of paths on windows tho.

Reply:
put back TODO comment.
I would rather try to support those formats too, but don't know how to test.
I've tried to connect my camera in MTP mode (what else would you connect ? :-)), but although it is shown in windows file manager as a device, it is NOT shown in DT in the native filechooser :-/